src/demokratie.md

```diff
@@ -375,37 +375,37 @@ verschiedenen Optionen in eine Reihenfolge bringen. Ich nehme an, dass beim
 systemischen Konsensieren etwas Ähnliches passieren muss. Vielleicht können
 beide Prozesse hier voneinander lernen, wenn sie nicht identisch sind.
 
-Ein weiteres, wichtiges Element muss die Möglichkeit von Minderheiten sein,
-dass nicht gegen ihre Interessen verstoßen wird. Sie muss also eine
+Ein weiteres wichtiges Element muss der Schutz von Minderheiten sein,
+sodass nicht gegen ihre Interessen verstoßen wird. Sie muss also eine
 Möglichkeit haben, um Schaden abzuwenden, ohne gleichzeitig Macht ausüben zu
 können, da sonst einseitige Entscheidungen zugunsten bestimmter Minderheiten
 getroffen werden könnten. Eine meiner Meinung nach gangbare Lösung wäre,
 dass man Minderheiten ein Veto-Recht gibt. Wie groß die Minderheit dabei sein
-muss, kann ich hier nicht sagen, das müsste ausgearbeitet werden.
+muss kann ich hier nicht sagen, das müsste ausgearbeitet werden.
 
 Da man gerade durch das Los-Verfahren der aleatorischen Demokratie kein
-beständiges Parlament hat, kann man im Falle eines (oder mehrerer) Vetos ein
-neues Konvent einberufen. Dadurch kann Reformstau effektiv verhindert werden,
+beständiges Parlament hat, kann man im Falle eines (oder mehrerer) Vetos einen
+neuen Konvent einberufen. Dadurch kann Reformstau effektiv verhindert werden,
 da sich nicht die gleichen Konstellationen in Konventen herausbilden können.
-Das Veto kann sowohl innerhalb eines Konvents angewandt werden (z. B. wenn es
-um die Wahl von Arbeitsgruppen innerhalb des Konvents geht) aber auch auf Ebene
+Das Veto kann sowohl innerhalb eines Konventes angewandt werden (z. B. wenn es
+um die Wahl von Arbeitsgruppen innerhalb des Konventes geht) aber auch auf Ebene
 der Abstimmungen (z. B. bei Volksabstimmungen). Ein neuer Konvent müsste dann
 auf die Einwände der Minderheit, die vom Veto-Recht Gebrauch gemacht hat,
 eingehen, da neue Optionen erarbeitet werden müssen oder zumindest eine neue
 öffentliche Debatte angeregt werden muss.
 
 Wahlen müssen dabei immer mit Papier ausgetragen werden. Software hat immer
 das Problem, dass kleine Fehler/Sicherheitslücken unbemerkt eine ganze Wahl
-manipulieren kann. Es ist daher wichtig, dass weiterhin freiwillige
+manipulieren können. Es ist daher wichtig, dass weiterhin freiwillige
 Wahlbeobachter ungehindert Zugang zum Wahlvorgang haben, um diesen unabhängig
 kontrollieren zu können. Idealerweise werden alle Wahlvorgänge aufgenommen
 und so zensiert, dass Personen nicht identifiziert werden können, der
-restliche Vorgang aber noch erkennbar. Diese Aufnahmen müssen der
+restliche Vorgang aber noch erkennbar ist. Diese Aufnahmen müssen der
 Öffentlichkeit zur Verfügung stehen, damit sich jeder Bürger selbst ein
 Bild machen kann, sollte es zu Wahlbetrugsvorwürfen kommen.
 [Hier](https://www.csail.mit.edu/news/mit-experts-no-dont-use-blockchain-vote)
 ist ein Artikel des MIT, in dem erklärt wird, warum auch neue Technologie wie
-Blockchain diese Probleme haben.
+Blockchain diese Probleme hat.
```

Comment:
Ich glaube hier ist "haben" korrekt und "Technologie" sollte im Plural sein, sonst wuerde ein Artikel fehlen

Reply:
Ok. In dem Text steht aber immer noch Technologie (Singular) bei mir.

Reply:
Ok. Bei mir steht aber immer noch Technologie (Singular) im Text.